litellm/proxy/auth/user_api_key_auth.py

```diff
@@ -105,39 +105,6 @@ def _get_bearer_token(
     return api_key
 
 
-def _is_ui_route_allowed(
```

Comment:
We should not run this 

This code allowed:
- non proxy admins to access all routes in `LiteLLMRoutes.ui_routes.value` 

We already have a function for checking this `RouteChecks.non_proxy_admin_allowed_routes_check`